blueprints/app/files/package.json

```diff
@@ -13,11 +13,12 @@
   "scripts": {
     "build": "ember build --environment=production",
     "lint": "npm-run-all --print-name --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
-    "lint:fix": "npm-run-all --print-name --aggregate-output --continue-on-error --parallel \"lint:*:fix\"",
+    "lint:fix": "npm-run-all \"lint:fix:script -- .\"",
```

Comment:
The `lint:fix` script now assumes every `lint:*:fix` script needs to run on the entire codebase because of the trailing `.`.

I'm not sure if that's an assumption we can make.
E.g. when you have a custom script like `"lint:json:fix": "prettier \"./**/*.json\" --write"`, instead of running `prettier \"./**/*.json\" --write"`, this will run `prettier \"./**/*.json\" --write" .` and will most likely fail / produce unexpected results as the trailing `.` supersedes `\"./**/*.json\"`.

Also:
- This removes the `--print-name` flag which was just added over in https://github.com/ember-cli/ember-cli/pull/9988
- `--` is not needed when using Yarn, using this results in the following warning:
   ```
   From Yarn 1.0 onwards, scripts don't require "--" for options to be forwarded. In a future version, any explicit "--" will be forwarded as-is to the scripts.
   ```

Reply:
a solution for this could be to make a `:script` version of the current `lint:js:fix` and `lint:hbs:fix` scripts which would take the files to run on as argument (with the `.` being the default used when `lint:js:fix` and `lint:hbs:fix` are executed) and then running `lint:*:fix:script` instead of `lint:*:fix`